packages/strapi-plugin-content-type-builder/admin/src/components/HeaderNav/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+*
+* HeaderNav
+*
+*/
+
+import React from 'react';
+import { Link } from 'react-router';
+import { join, map, take } from 'lodash';
+import EditForm from 'components/EditForm';
+import List from 'components/List';
+import styles from './styles.scss';
+
+class HeaderNav extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  render() {
+    const baseUrl = join(take(this.props.path.split('/'), 4), '/');
+    const component = this.props.renderListComponent ? <List {...this.props} /> : <EditForm {...this.props} />;
+    return (
+      <div className={styles.headerNav}>
+        <div className={`${styles.noPaddingLeft} container-fluid `}>
+          <div className="row">
+            <div className="col-md-12">
+              <div className={styles.headerContainer}>
+                {map(this.props.links, (link, key) => {
+                  const notifActive = link.active ? <div className={styles.notifPoint} /> : '';
+                  return (
+                    <Link key={key} className={styles.headerLink} to={`${baseUrl}/${link.name}`} activeClassName={styles.linkActive}>
+                      <div></div>
```

Comment:
Is this necessary?

Reply:
Yes, BTW it's a component from the settings-manager